backend/telemetry_core/src/aggregator/inner_loop.rs

```diff
@@ -55,13 +55,13 @@ pub enum FromShardWebsocket {
     Add {
         local_id: ShardNodeId,
         ip: std::net::IpAddr,
-        node: common::node_types::NodeDetails,
+        node: Box<common::node_types::NodeDetails>,
```

Comment:
`std::mem::size_of::<NodeDetails>() == 111`

I think that is small enough to avoid to `Box` this one

Reply:
Yeah, but enum variant is around 369 bytes:
```
warning: large size difference between variants
  --> telemetry_core/src/aggregator/inner_loop.rs:48:1
   |
48 |   / pub enum FromShardWebsocket {
49 |   |     /// When the socket is opened, it'll send this first
50 |   |     /// so that we have a way to communicate back to it.
51 |   |     Initialize {
...    |
55 | / |     Add {
56 | | |         local_id: ShardNodeId,
57 | | |         ip: std::net::IpAddr,
58 | | |         node: common::node_types::NodeDetails,
59 | | |         genesis_hash: common::node_types::BlockHash,
60 | | |     },
   | |_|_____- the largest variant contains at least 369 bytes
61 |   |     /// Update/pass through details about a node.
62 | / |     Update {
63 | | |         local_id: ShardNodeId,
64 | | |         payload: Box<node_message::Payload>,
65 | | |     },
   | |_|_____- the second-largest variant contains at least 16 bytes
...    |
69 |   |     Disconnected,
70 |   | }
   |   |_^ the entire enum is at least 376 bytes
   |
   = note: `#[warn(clippy::large_enum_variant)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
help: consider boxing the large fields to reduce the total size of the enum
   |
58 |         node: Box<common::node_types::NodeDetails>,
   |               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
````